components/Dropdown.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styles from "../styles/Home.module.scss";
+import "../styles/dropdown.module.scss";
+import DropdownMenu from '@radix-ui/react-dropdown-menu';
```

Comment:
This package doesn't seem to be compatible with React 18. We can merge this I think, but long term we will probably have to find a different dropdown library.